packages/web/components/Landing/WhatDo.tsx

```diff
@@ -0,0 +1,234 @@
+import { Box, Container, ListItem, Text, UnorderedList } from '@metafam/ds';
+import BackgroundImage from 'assets/landing/whatdo/computer.png';
+import { FullPageContainer } from 'components/Container';
+// import { gsap } from "gsap";
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import { useRef } from 'react';
+
+import { LandingNextButton } from './LandingNextButton';
+// import { LandingNextButton } from './LandingNextButton';
+
+export const WhatDo: React.FC = () => {
+  const ref = useRef<HTMLDivElement>(null);
+  // const slidesContainer = useRef<HTMLDivElement>(null);
+  // const slides = useRef<Array<HTMLDivElement>>([]);
+  const onScreen = useOnScreen(ref);
+  const section = 'what-do';
+
+  // const createSlidesRefs = (slide: HTMLDivElement, index: number) => {
+  //   slides.current[index] = slide
+  //   console.log('slides', slides);
+
+  // }
+
+  // useEffect(() => {
+  //   const totalSlides = slides.current.length
+
+  //   gsap.to(slides.current, {
+  //     xPercent: 0 * (totalSlides - 1),
+  //     ease: 'none',
+  //     scrollTrigger: {
+  //       trigger: slidesContainer.current,
+  //       pin: true,
+  //       scrub: 1,
+  //       snap: 1 / (totalSlides - 1),
+  //       end: () => `+=${slidesContainer.current?.offsetWidth}`
+  //     }
+  //   })
+  // }, []);
```

Comment:
more commented out code. why not just delete it?

Reply:
lol kept for possible future use but kk I'll delete. 😄

Reply:
well, stuff like that might be useful. if you decide to leave it in, it would imo be nice to write a short explanation of why it's still there and what was/could be the purpose of it.

Reply:
For sure. Will do in future but I've now removed it. :)